test/e2e/garbage_collector.go

```diff
@@ -55,23 +91,41 @@ func newOwnerRC(f *framework.Framework, name string) *v1.ReplicationController {
 		Spec: v1.ReplicationControllerSpec{
 			Replicas: &replicas,
 			Selector: map[string]string{"app": "gc-test"},
-			Template: &v1.PodTemplateSpec{
-				ObjectMeta: v1.ObjectMeta{
-					Labels: map[string]string{"app": "gc-test"},
-				},
-				Spec: v1.PodSpec{
-					Containers: []v1.Container{
-						{
-							Name:  "nginx",
-							Image: "gcr.io/google_containers/nginx:1.7.9",
-						},
-					},
-				},
-			},
+			Template: &podTemplateSpec,
 		},
 	}
 }
 
+// verifyRemainingDeploymentsAndReplicaSets verifies if the number of the remaining deployment
+// and rs are deploymentNum and rsNum. It returns error if the
+// communication with the API server fails.
+func verifyRemainingDeploymentsAndReplicaSets(
+	f *framework.Framework,
+	clientSet clientset.Interface,
+	deployment *v1beta1.Deployment,
+	deploymentNum, rsNum int,
+) (bool, error) {
+	rs, err := clientSet.Extensions().ReplicaSets(f.Namespace.Name).List(v1.ListOptions{})
+	if err != nil {
+		return false, fmt.Errorf("Failed to list rs: %v", err)
+	}
+
+	var ret = true
```

Comment:
nit^2: move this to the start of the function, remove the newline, so that the code looks symmetric.